src/test/java/com/epam/tc/hw7/ex1/pages/MetalColorPage.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.tc.hw7.ex1.pages;
+
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.tc.hw7.ex1.TestData;
+import java.util.Arrays;
+import java.util.List;
+
+@Url("/metals-colors.html")
+@Title("Metal and Colors")
+public class MetalColorPage extends WebPage {
```

Comment:
Check class work or JDI templates. 
hierarchy of JDI's PO should be like this:
Site -> Page -> Form -> Elements